### ISSUE TITLE
[Bug]:当我点击提示与技巧时，弹出窗口提示“Uri was not launched”

### ISSUE THREAD

**Issue by SilverwolfError** [TranslucentTB/TranslucentTB#995]
### Please check for a duplicate first

- [x] I have checked the issue tracker if my bug has already been fixed or answered in another (possibly closed) issue.

### Steps to reproduce the issue

点击TranslucentTB图标，点击“提示与技巧”，弹出窗口提示“Uri was not launched”。
![image](https://github.com/TranslucentTB/TranslucentTB/assets/155169669/6b4be853-a62f-4542-aa20-d0816fef332b)

### Expected results

我希望能找到一些有用的关于TranslucentTB的技巧

### Actual results

但实际却给我弹出了一个窗口，提示“Uri was not launched.”

### Windows version

Windows 11 SV2 (builds 22621-22622)

### Relevant log output

```shell
[2024-03-16 10:11:27.474] [error] [uwp.cpp:85] Uri was not launched.
```

**Comment by Whitehachi** [comment]
opened返回了False才会这样，我打开没有问题。有可能是浏览器的问题...

**Comment by sylveon** [comment]
Yes, this is reported when Windows signals to the app that it failed opening an URL - something is probably wrong with your default web browser

**Comment by Whitehachi** [comment]
也就是说，指定的网页浏览器出现了问题？
我默认的网页浏览器是Microsoft Edge(msedge.exe)，并未出现任何错误。
